EventFiltering/PWGHF/HFFilterHelpers.h

```diff
@@ -35,19 +35,25 @@
 #include "Math/Vector4D.h"
 
 #include "CCDB/CcdbApi.h"
-#include "DataFormatsParameters/GRPMagField.h"
-#include "DataFormatsParameters/GRPObject.h"
+#include "CCDB/BasicCCDBManager.h"
 #include "DataFormatsTPC/BetheBlochAleph.h"
-#include "DetectorsBase/Propagator.h"
 #include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
 #include "Framework/DataTypes.h"
 #include "Framework/HistogramRegistry.h"
+#include "Framework/AnalysisHelpers.h"
+#include "Framework/O2DatabasePDGPlugin.h"
 
 #include "Common/Core/RecoDecay.h"
 #include "Common/Core/trackUtilities.h"
 #include "PWGHF/DataModel/CandidateReconstructionTables.h"
 #include "PWGHF/DataModel/CandidateSelectionTables.h"
 
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::constants::math;
+
```

Comment:
```suggestion
```
Using directives should never appear in headers.

Reply:
Done, thanks. For my education, is it for memory consumption or is there any other reason?